src/GPS/GPSManager.cc

```diff
@@ -24,6 +24,49 @@ GPSManager::GPSManager(QGCApplication* app, QGCToolbox* toolbox)
 GPSManager::~GPSManager()
 {
     disconnectGPS();
+    if(_gpsRtkFactGroup)
+    {
+        delete _gpsRtkFactGroup;
+    }
```

Comment:
`delete` on `nullptr` is already a no-op:

```suggestion
    delete _gpsRtkFactGroup;
```